vividus-extension-selenium/src/main/java/org/vividus/selenium/cloud/AbstractCloudTestStatusManager.java

```diff
@@ -78,6 +83,23 @@ public final void updateCloudTestStatus(BeforeWebDriverQuitEvent event)
         }
     }
 
+    @Subscribe
+    public final void waitUntilTestComplete(AfterWebDriverQuitEvent event)
+    {
+        waitUntilCloudTestIsCompleted(event.sessionId());
+    }
+
+    private void waitUntilCloudTestIsCompleted(String sessionId)
```

Comment:
Question: default wait duration? 30 sec? right now it takes 2-4 sec to complete test

Reply:
20 seconds